eng/common/scripts/update-docs-ci.ps1

```diff
@@ -59,7 +59,7 @@ function UpdateParamsJsonPython($pkgs, $ciRepo, $locationInDocRepo){
   foreach ($releasingPkg in $pkgs) {
     if ($visibleInCI.ContainsKey($releasingPkg.PackageId)) {
       $packagesIndex = $visibleInCI[$releasingPkg.PackageId]
-      $existingPackageDef = $targetData[$packagesIndex]
+      $existingPackageDef = $allJson.packages[$packagesIndex]
```

Comment:
Please ensure this change makes it into the tools repo before the next sync.

Reply:
That's a given 👍